### ISSUE TITLE
Optional field documentation parsing is broken

### ISSUE THREAD

**Issue by CzB404** [ndsev/zserio#71]
Zserio does not generate documentation text for the following script:
```
struct A
{
    /**
     * Docstring.
     */
    optional varint x;
};
```

But it does for this:
```
struct A
{
    optional
    /**
     * Docstring.
     */
    varint x;
};
```

**Comment by mikir** [comment]
Thanks for that. This is a bug, we will fix it soon to be in the next release.

**Comment by Mi-La** [comment]
Closed by  9be4be1.
